ethereum/referencetests/src/reference-test/java/org/hyperledger/besu/ethereum/core/TransactionTest.java

```diff
@@ -125,8 +130,30 @@ public void london(final String name, final TransactionTestCaseSpec spec) {
     milestone(spec, name, "London", new LondonGasCalculator(), Optional.of(Wei.of(0)));
   }
 
+  @ParameterizedTest(name = "Name: {0}")
+  @MethodSource("getTestParametersForConfig")
+  public void merge(final String name, final TransactionTestCaseSpec spec) {
+    milestone(spec, name, "Merge", new LondonGasCalculator(), Optional.of(Wei.of(0)));
+  }
+
+  @ParameterizedTest(name = "Name: {0}")
+  @MethodSource("getTestParametersForConfig")
+  public void shanghai(final String name, final TransactionTestCaseSpec spec) {
+    milestone(spec, name, "Shanghai", new ShanghaiGasCalculator(), Optional.of(Wei.of(0)));
+  }
+
+  @ParameterizedTest(name = "Name: {0}")
+  @MethodSource("getTestParametersForConfig")
+  public void cancun(final String name, final TransactionTestCaseSpec spec) {
+    milestone(spec, name, "Cancun", new CancunGasCalculator(), Optional.of(Wei.of(0)));
```

Comment:
I couldn't find Cancun TransactionTests in [ethereum/tests](https://github.com/ethereum/tests)#develop, are you adding this now so we don't forget them once they get added as it happened with Merge and Shangai?

Reply:
Adding so we don't forget.  If blob TXes are ever added to reference tests we will need cancun.